index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <h3 class="logo">My Logo</h3>
+        </div>
+        <div class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </div>
+    </header>
```

Comment:
- [ ] Kindly make sure the header has correct spacing to the right side, left side and at the top as shown in the video.

https://user-images.githubusercontent.com/70269507/170498627-c69ee10c-164a-4e9c-9ff6-76109608f63a.mov